app/src/assets/localization/en/pipette_wizard_flows.json

```
@@ -1,15 +1,17 @@
 {
   "before_you_begin": "Before you begin",
-  "remove_labware_to_get_started": "<block>To get started, remove labware from the rest of the deck and clean up the work area to make attachment and calibration easier. Also gather the needed equipment shown on the right hand side</block><block>The calibration probe is included with the robot and should be stored on the right hand side of the door opening.</block>",
-  "attach_probe": "Attach Calibration Probe",
-  "install_probe": "Take the calibration probe from its storage location. Make sure its latch is in the unlocked (straight) position. Press the probe firmly onto the A1 (back left corner) pipette nozzle and then lock the latch. Then test that the probe is securely attached by gently pulling it back and forth.",
+  "remove_labware_to_get_started": "<block>To get started, remove labware from the deck and clean up the working area to make calibration easier. Also gather the needed equipment shown to the right.</block><block>The calibration probe is included with the robot and should be stored on front pillar of the robot.</block>",
+  "attach_probe": "attach calibration probe",
+  "install_probe": "Take the calibration probe from its storage location. Make sure its latch is in the unlocked (straight) position. Press the probe firmly onto the pipette nozzle and then lock the latch. Then test that the probe is securely attached by gently pulling it back and forth.",
   "install_probe_8_channel": "<block>Take the calibration probe from its storage location. Make sure its latch is in the unlocked (straight) position. Press the probe firmly onto the <strong>backmost</strong> pipette nozzle and then lock the latch. Then test that the probe is securely attached by gently pulling it back and forth.</block>",
+  "install_probe_96_channel": "<block>Take the calibration probe from its storage location. Make sure its latch is in the unlocked (straight) position. Press the probe firmly onto the A1 <strong>(back left corner)</strong> pipette nozzle and then lock the latch. Then test that the probe is securely attached by gently pulling it back and forth.</block>",
   "begin_calibration": "Begin calibration",
-  "remove_cal_probe": "Remove Calibration Probe",
-  "remove_probe": "Unlatch the calibration probe, remove it from the pipette nozzle, and return it to its storage location.",
+  "remove_cal_probe": "remove calibration probe",
+  "remove_probe": "unlatch the calibration probe, remove it from the  nozzle, and return it to its storage location.",
```

```suggestion
  "remove_probe": "unlatch the calibration probe, remove it from the nozzle, and return it to its storage location.",
```
double space, was probably left in due to a sneaky soft wrap

---

omg i was trying to figure out why when testing this string, it kept failing and it is because i missed that there was the double space. thanks!